examples/player/simid_player.html

```diff
@@ -200,6 +200,7 @@
         <button onclick="switchCreative('extender.html')">extender</button>
         <button onclick="switchCreative('testers_nonlinear.html')">testers_nonlinear</button>
         <button onclick="switchCreative('banner_nonlinear.html')">banner_nonlinear</button>
+        <button onclick="switchCreative('hover_nonlinear.html')">hover_nonlinear</button>
```

Comment:
it would be cool if you added the defaults to switchCreative for this one.  Like switching the radio buttons to nonlinear and adding some default dimensions that demonstrate this correctly.

Reply:
the default dimensions do demonstrate this correctly. I can definitely try switching the radio button to nonlinear